src/hacking/using-devtools.md

```diff
@@ -0,0 +1,64 @@
+# DevTools
+
+[Firefox DevTools](https://firefox-source-docs.mozilla.org/devtools-user) are a set of web developer tools that can be used to examine, edit, and debug a website's HTML, CSS, and JavaScript.
+Servo has support for a subset of DevTools functionality, allowing for simple debugging.
+
+## Connecting to Servo
+
+1. Run servoshell with the DevTools server enabled.
+   Use the same port as before.
```

Comment:
this is the first mention of a port on this page. could you please clarify?

Reply:
Sorry about that. I reordered the steps at one point to fit better with the screenshot and that was left behind. I clarified it in the last commit, hopefully it is more clear now.